test/unit/com/unifina/api/UpdateProductCommandSpec.groovy

```diff
@@ -195,13 +197,26 @@ class UpdateProductCommandSpec extends Specification {
 			description: "new description",
 			pendingChanges: '''{"name":"new name","description":"new description"}'''
 		)
-		command.permissionService = Mock(PermissionService)
 		product.pricePerSecond = 5
 
 		when:
-		command.updateProduct(product, new SecUser())
+		command.updateProduct(product, new SecUser(), permissionService)
 		then:
-		1 * command.permissionService.canShare(_, product) >> false
+		1 * permissionService.canShare(_, product) >> false
 		thrown(FieldCannotBeUpdatedException)
 	}
+
+	void "updateProduct() doesn't check sharing permission when pendingChanges is not given"() {
```

Comment:
This test is good to have too, but in my request for an additional test, I meant the case where:
- `canShare(_, product) >> true`
- check that the `pendingChanges` field was updated

Reply:
I had this open so went ahead and committed this